src/api/api_pass.cc

```diff
@@ -66,6 +66,11 @@ TVM_REGISTER_API("ir_pass.Equal")
     }
   });
 
+TVM_REGISTER_API("ir_pass.StorageFlatten")
+.set_body([](TVMArgs args, TVMRetValue *ret) {
+      *ret = StorageFlatten(args[0], args[1], args[2],
+                            args.size() == 3 ? false : args[3]);
```

Comment:
The following would be better (to avoid duplicating the default value in two places)
```
if (args.size() <= 3) {
        *ret = StorageFlatten(args[0], args[1], args[2]);
} else {
        *ret = StorageFlatten(args[0], args[1], args[2], args[3]);
}
```

Reply:
Thanks for review, updated.